Algorithm/QCAlgorithm.Universe.cs

```diff
@@ -449,7 +449,7 @@ public void AddUniverse(SecurityType securityType, string name, Resolution resol
         /// </summary>
         /// <param name="security">The security to add</param>
         /// <param name="configurations">The <see cref="SubscriptionDataConfig"/> instances we want to add</param>
-        private void AddToUserDefinedUniverse(
+        protected void AddToUserDefinedUniverse(
```

Comment:
Because `public Security AddSecurity(SecurityType securityType, string ticker, ...` takes a string ticker (assuming that it is the most up-to-date value), we'd need to map the ticker ourselves and provide it to the `AddSecurity` method. For the sake of fast development and to reduce potential bugs, we chose to make this method accessible.